src/Microsoft.Graph/Microsoft.Graph.csproj

```diff
@@ -58,6 +58,6 @@ July 2018 Release Summary (version 1.10.0)
     <PackageReference Include="Newtonsoft.Json" Version="[6.0.1,12)" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.1' ">
-    <PackageReference Include="Newtonsoft.Json" Version="[9.0.1,12)" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
Why did you remove the upper bounds limit on JSON.net? The reason we had that there was because we wanted to make sure that any breaking change in JSON.net didn't affect us before we built targeting the new major version. This does make it so we need to update our dependency when JSON.net upgrades. It would be nice if we implemented some CI so that when the dependency updates, we build and run tests with the updated depencency.